.pre-commit-config.yaml

```diff
@@ -70,15 +70,13 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
-        name: Run isort to sort imports in Python files
+        name: Run isort for python-sdk
```

Comment:
@feluelle Any reason we are doing this separately for Python/SQL SDK?

Reply:
Good question. I don't know. 🤷  @dimberman could you help us please?